appserver/tests/appserv-tests/devtests/admin/cli/run_test.sh

```diff
@@ -96,6 +96,5 @@ case $OPT in
 	list_test_ids )
 		list_test_ids;;
 	run_test_id )
-		trap post_test_run EXIT
```

Comment:
I am not sure why this is appearing. I have not done any changes in this line. I have to check it and get back.